app/views/shared/_user_nav.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="navbar-fixed">
+    <nav>
+      <div class="nav-wrapper">
+        <%= link_to "Folio Share", home_path, class: "brand-logo" %>
+        <ul class="right hide-on-med-and-down">
+          <li><%= link_to current_user.full_name, user_path(current_user) %></li>
+          <li><%= link_to "Edit Account Details", edit_user_path(current_user) %></li>
```

Comment:
I vote to move this link to the user's account page instead of on the navbar. Personally, I like a navbar that's as minimalist as possible.